File: Oops_concepts/oops.py
class person:
    def set_person(self,age,name):
        self.name=name
        self.age=age
    def print_person(self):
        print(self.name)
        print(self.age)

obj=person()
obj.set_person(23,'john')
obj.print_person()

File: htmlportion/javascript/oops_concept/oops.js
class student{
    constructor(name,age){
        this.name=name;
        this.age=age;
    }

    printstud(){
        console.log(this.name)
        console.log(this.age);
    }

}

var obj=new student("bibin",24)

obj.printstud()

